src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  List,
+  SxProps,
+  Theme
+} from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+import { TutorProfileProps } from '~/pages/edit-profile/EditProfile.constants'
+import { TutorProfileTabsEnum } from '~/types'
+
+interface SidebarMenu {
+  tabsData: TutorProfileProps
+  handleClick: (tab: TutorProfileTabsEnum) => void
+  styles?: SxProps<Theme>
+}
+
+const SidebarMenu = ({ handleClick, tabsData, styles }: SidebarMenu) => {
```

Comment:
I think we use `FC` and pass there generic for types. What do you think ?

Reply:
Agree with you